lib/fizzy/execute.cpp

```diff
@@ -677,7 +676,8 @@ std::optional<uint32_t> find_export(const Module& module, ExternalKind kind, std
 
 std::unique_ptr<Instance> instantiate(Module module,
     std::vector<ExternalFunction> imported_functions, std::vector<ExternalTable> imported_tables,
-    std::vector<ExternalMemory> imported_memories, std::vector<ExternalGlobal> imported_globals)
+    std::vector<ExternalMemory> imported_memories, std::vector<ExternalGlobal> imported_globals,
+    unsigned memory_pages_limit /* = DefaultMemoryPagesLimit */)
```

Comment:
```suggestion
    unsigned memory_pages_limit /*= DefaultMemoryPagesLimit*/)
```
To be consistent with `/*unused*/`?